src/routes/add-custom-discipline/__tests__/AddCustomDisciplineScreen.spec.tsx

```diff
@@ -84,4 +89,14 @@ describe('AddCustomDisciplineScreen', () => {
     fireEvent.press(submitButton)
     expect(await findByText(labels.addCustomDiscipline.error.wrongCode)).not.toBeNull()
   })
+
+  it('should open qr code scanner', async () => {
+    const { findByLabelText } = render(<AddCustomDisciplineScreen navigation={navigation} />, {
+      wrapper: wrapWithTheme
+    })
+    const QRCodeIcon = await findByLabelText('qr-code-scanner')
```

Comment:
All tests in this file are async, because this component is not rendered until `useReadCustomDisciplines` returns the customDisciplines from AsyncStorage.